### ISSUE TITLE
Add translations for terra-paginator

### ISSUE THREAD

**Issue by saedar** [cerner/terra-core#1493]
# Issue Description
We need to add support for translations in the terra-paginator component.

## Issue Type
- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Proper translated text replaces the existing text.

Example: "Page 1 of 44" -> the equivalent in French.

## Current Behavior
It is statically English.

**Comment by emilyrohrbough** [comment]
@saedar Were translations requested?

**Comment by saedar** [comment]
@emilyrohrbough It was something that came up during the review of my initial implementation. Basically, it would be good to enhance our accessibility by providing i18n for the languages we support. No specific requests that I am aware of.

**Comment by yuderekyu** [comment]
We should send a request to the translation team then, right?

**Comment by saedar** [comment]
Requested translations with the following keys:
```
{
  "Terra.paginator.first": "First",
  "Terra.paginator.last": "Last",
  "Terra.paginator.next": "Next",
  "Terra.paginator.pagePosition": "Page {currentPage} of {totalPages}",
  "Terra.paginator.previous": "Previous",
}
```

**Comment by yuderekyu** [comment]
Did we ever get these translations?

**Comment by neilpfeiffer** [comment]
Will be resolved with #2485
